patch/driver-ingrasys-eeprom.patch

```diff
@@ -0,0 +1,269 @@
+Add Ingrasys S9100 Mother Board EEPROM
```

Comment:
Is that out-of-treee module package like [sonic-platform-modules-s6000](https://github.com/Azure/sonic-platform-modules-s6000)?